### PR TITLE
DBZ-IFX-1 Completed dealing with truncate.

### PR DESCRIPTION
**Pull request by ccrocc**
1. Handle truncate in Streaming Change Event.
2. Integration test with truncate event.

### CODE REVIEW

File: src/main/java/laoflch/debezium/connector/informix/InformixStreamingChangeEventSource.java [hunk 1]
```diff
@@ -15,12 +15,7 @@
 
 import com.informix.jdbc.IfmxReadableType;
 import com.informix.stream.api.IfmxStreamRecord;
-import com.informix.stream.cdc.records.IfxCDCBeginTransactionRecord;
-import com.informix.stream.cdc.records.IfxCDCCommitTransactionRecord;
-import com.informix.stream.cdc.records.IfxCDCMetaDataRecord;
-import com.informix.stream.cdc.records.IfxCDCOperationRecord;
-import com.informix.stream.cdc.records.IfxCDCRollbackTransactionRecord;
-import com.informix.stream.cdc.records.IfxCDCTimeoutRecord;
+import com.informix.stream.cdc.records.*;
```
Comment (leoncamel) [inline src/main/java/laoflch/debezium/connector/informix/InformixStreamingChangeEventSource.java]: Please follow the debezium community's [checkstyle](https://github.com/debezium/debezium/blob/main/support/checkstyle/src/main/resources/checkstyle.xml#L24).

We should import the specific classes and avoid importing star.

File: src/test/java/laoflch/debezium/connector/informix/informixCdcTruncateIT.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import static laoflch.debezium.connector.informix.InformixConnectorConfig.SNAPSHOT_MODE;
+import static laoflch.debezium.connector.informix.InformixConnectorConfig.SnapshotMode.INITIAL_SCHEMA_ONLY;
+import static org.fest.assertions.Assertions.assertThat;
+
+public class informixCdcTruncateIT extends AbstractConnectorTest {
```
Comment (leoncamel) [inline src/test/java/laoflch/debezium/connector/informix/informixCdcTruncateIT.java]: Following the java common coding standard, the class name of the first letter is uppercase.

**Comment by leoncamel** [comment]
@ccrocc Can you review these comments?

**Review by leoncamel — CHANGES_REQUESTED** [review]
Please polish these minor changes. Thanks.

File: src/main/java/laoflch/debezium/connector/informix/InformixStreamingChangeEventSource.java (hunk 1 above)
Comment (ccrocc) [reply src/main/java/laoflch/debezium/connector/informix/InformixStreamingChangeEventSource.java]: Resolved. 
Avoid importing star and Avoid using the import optimization of IDE in feature.

File: src/test/java/laoflch/debezium/connector/informix/informixCdcTruncateIT.java (hunk 2 above)
Comment (ccrocc) [reply src/test/java/laoflch/debezium/connector/informix/informixCdcTruncateIT.java]: Careless spelling, fixed. Thx~

File: src/test/java/laoflch/debezium/connector/informix/informixCdcTruncateIT.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
+package laoflch.debezium.connector.informix;
```
Comment (leoncamel) [inline src/test/java/laoflch/debezium/connector/informix/informixCdcTruncateIT.java]: Please add the missing file header.

**Pull request merged**